central/notifiers/cscc/cscc_test.go

```diff
@@ -0,0 +1,87 @@
+package cscc
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/golang/mock/gomock"
+	clusterMocks "github.com/stackrox/rox/central/cluster/datastore/mocks"
+	"github.com/stackrox/rox/central/notifiers/cscc/findings"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWithFakeCSCC(t *testing.T) {
+
+	const (
+		projectKey = "FJ"
+	)
+
+	sourceID := "organizations/847401270788/sources/394194234514334994"
+
+	s := &storage.Notifier{
+		Name:         "FakeSCC",
+		UiEndpoint:   "https://central.stackrox",
+		Type:         "scc",
+		LabelDefault: projectKey,
+		Config: &storage.Notifier_Cscc{
+			Cscc: &storage.CSCC{
+				ServiceAccount: "test_service_account",
+				SourceId:       sourceID,
+			},
+		},
+	}
+
+	cluster := &storage.Cluster{
+		Id:   "test_id",
+		Name: "test_name",
+		Status: &storage.ClusterStatus{
+			ProviderMetadata: &storage.ProviderMetadata{
+				Zone: "test_zone",
+				Provider: &storage.ProviderMetadata_Google{
+					Google: &storage.GoogleProviderMetadata{
+						Project:     "test_project",
+						ClusterName: "test_cluster",
+					},
+				},
+			},
+		},
+	}
+
+	mockCtrl := gomock.NewController(t)
+	clusterStore := clusterMocks.NewMockDataStore(mockCtrl)
+	clusterStore.EXPECT().GetCluster(gomock.Any(), "test_id").Return(cluster, true, nil)
+	scc, _ := newCSCC(s, clusterStore)
+
+	alertID := "myAlertID"
+	severity := findings.High
+
+	testAlert := &storage.Alert{
+		Id: alertID,
+		Policy: &storage.Policy{
+			Id:             "myPolicyID",
+			Name:           "myPolicy",
+			Description:    "Fake policy",
+			PolicySections: []*storage.PolicySection{},
+			Severity:       storage.Severity_HIGH_SEVERITY,
+		},
+		Entity: &storage.Alert_Deployment_{Deployment: &storage.Alert_Deployment{
+			Name:      "myDeployment",
+			Id:        "myDeploymentID",
+			ClusterId: "test_id",
+		}},
+		Time: types.TimestampNow(),
+	}
+	findingID := ""
+	var finding *findings.Finding
+	var err error
+	findingID, finding, err = scc.InitFinding(context.Background(), testAlert)
+	assert.Equal(t, nil, err)
```

Comment:
Use `assert.NoError` instead for error checking.

Reply:
done